File: LumeLaht_RoomApi.Application/Dto/CreateRoomRequest.cs
namespace LumeLaht_RoomApi.Application.Dto
{
    public class CreateRoomRequest
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal PricePerHour { get; set; }
        public string Status { get; set; }
        public Guid AddressId { get; set; }
        public List<Guid>? ActivityIds { get; set; }
    }
}

File: frontend/src/types/roomTypes/CreateRoomRequest.ts
import { AddressResponse } from './AddressResponse';
import { ActivityResponse } from './ActivityResponse';
import { RoomImages } from './RoomImages';
export interface CreateRoomRequest {
    name?: string | undefined;
    description?: string | undefined;
    pricePerHour?: number;
    status?: string;
    addressId?: string;
    address?: AddressResponse;
    activities?: ActivityResponse[] | undefined;
    roomImages?: RoomImages[] | undefined;
}
